controllers/postgresqldatabase_controller.go

```diff
@@ -293,12 +291,18 @@ func (r *PostgreSQLDatabaseReconciler) adminCredentials(ctx context.Context, par
 	// empty, then return credentials stored in the corresponding
 	// `PostgreSQLHostCredentials` resource.
 	if host == "" && params.hostCredentials != "" {
-		return r.remoteCredentials(ctx, params)
+		reqLogger.Info(fmt.Sprintf("Using remote host credential from PostgreSQLHostCredentials resource %s/%s", params.namespace, params.hostCredentials))
```

Comment:
Just making sure that we do not leak any credentials from `params.hostCredentials`?

Reply:
I'll have to check. Good point. On top of my head I expect this to be a type of `HostCredential` which hides it secrets when serialized :)

Reply:
Just double checked. I was wrong, but we are still good. It contains the name of the `PostgreSQLHostCredentials` resource referenced, so nothing secret in there.